extensions/amp-recaptcha-input/0.1/test/test-amp-recaptcha-input.js

```diff
@@ -32,19 +33,24 @@ describes.realWin('amp-recaptcha-input', {
     toggleExperiment(win, 'amp-recaptcha-input', true);
   });
 
-  function getRecaptchaInput(skipSiteKey, skipAction) {
-    const ampRecaptchaInput = doc.createElement('amp-recaptcha-input');
-    ampRecaptchaInput.setAttribute('layout',
-        'nodisplay');
-    if (!skipSiteKey) {
-      ampRecaptchaInput.setAttribute('data-sitekey',
-          'fake-sitekey-fortesting');
-    }
-    if (!skipAction) {
-      ampRecaptchaInput.setAttribute('data-action',
-          'unit-testing');
-    }
+  const ampRecaptchaInputAttributes = {
+    layout: 'nodisplay',
+    name: 'recaptcha-test',
+    'data-sitekey': 'fake-sitekey-fortesting',
```

Comment:
nit: since you're using string key, can we convert to 'layout' and 'name', just to match style : )